### ISSUE TITLE
Class constructor Loader cannot be invoked without 'new'

### ISSUE THREAD

**Issue by ffdead** [pmndrs/react-xr#48]
Hello!

Getting this error with version `3.0.1`. Error disappears when rolling back to `3.0.0`.

![](https://www.dropbox.com/s/v9ijqbiwxkyye03/Screenshot%202021-04-30%20at%2008.01.12.png?dl=0&raw=1)

Deps:
```
    "@react-spring/three": "9.1.2",
    "@react-three/drei": "4.3.2",
    "@react-three/fiber": "6.0.17",
    "@react-three/xr": "3.0.1",
    "prop-types": "15.7.2",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "react-scripts": "4.0.3",
    "three": "0.128.0"
  ```

**Comment by AndrewPrifer** [comment]
Looks like the offending code is actually in three-stdlib, which explains why the error occurs from 3.0.0 to 3.0.1.

**Comment by joshuaellis** [comment]
You need to be running a version of `three-stdlib@>=2.0.1`. What's weird is that version of `drei` should have the correct version with it. have you tried a clean install?

**Comment by AndrewPrifer** [comment]
@joshuaellis the error comes from xr's own stdlib dependency, which is fixed in the ^1.x range.

**Comment by sniok** [comment]
Hi, I removed three-stdlib dependency for now, we only use a couple of files from there so it doesn't make sense to have an external dependency just for that. This should be fixed in 3.1.0

**Comment by CodyJasonBennett** [comment]
Unless we'll want to leave a warning for older versions, this seems to be resolved.
